ste/jobStatusManager.go

```diff
@@ -106,6 +114,10 @@ func (jm *jobMgr) handleStatusUpdateMessage() {
 			// There is no need to keep sending the same items over and over again
 			js.FailedTransfers = []common.TransferDetail{}
 			js.SkippedTransfers = []common.TransferDetail{}
+
+		case <-jstm.done:
+			fmt.Println("Cleanup JobStatusmgr")
```

Comment:
Since we're already using loggers, may as well use them instead. We avoid using fmt.Println().
AzCopy has quiet-mode which doesn't print anything on the console and fmt.Print statements can create noise on terminal/